tests/test_featureset.py

```diff
@@ -845,3 +846,68 @@ def test_convert_featureset():
                                                           '.csv', '.arff',
                                                           '.libsvm'], 2):
         yield check_convert_featureset, from_suffix, to_suffix
+
+
+def featureset_creation_from_dataframe_helper(with_labels):
+    """
+    Helper function for the two unit tests for FeatureSet.from_data_frame().
+    Since labels are optional, run two tests, one with, one without.
```

Comment:
Ok 👍🏼  I don't know much about vectorizers, honestly.  Aren't there quite a few of them one could use?